lib/analytics/reddit-analyzer.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
```

Comment:
I think I'm going to start using it by default - so I'll leave out a comment.

But in case you're wondering, it does the following (stealing from MDN):
 * strict mode eliminates some JavaScript silent errors by changing them to throw errors.
 * strict mode fixes mistakes that make it difficult for JavaScript engines to perform optimizations: strict mode code can sometimes be made to run faster than identical code that's not strict mode. 
 * strict mode prohibits some syntax likely to be defined in future versions of ECMAScript.

Mozilla's documentation is very gewd: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode/Transitioning_to_strict_mode